dev-util/vulkan-tools/vulkan-tools-1.2.189.0.ebuild

```diff
@@ -13,9 +13,9 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_SUBMODULES=()
 	inherit git-r3
 else
-	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/sdk-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
There's a `sdk-1.2.189.1` tag in Vulkan-Tools. Should we go for that instead of .0?

I'd prefer a version bump to .1 rather than changing a stable ebuild's distfile.